src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -147,10 +148,28 @@
       + " mapping conflict or a field name containing illegal characters. Valid options are "
       + "'ignore', 'warn', and 'fail'.";
 
+  public static final String CONNECTION_SSL_CONFIG = "elastic.https";
+  private static final String CONNECTION_SSL_DOC = "Override to specify whether to use SSL "
+      + "connection in all cases. By default, the connector will use SSL when at "
+      + "least one URL begins with 'https:'. This config allows users to specify SSL even if "
+      + "host is specified without a protocol.";
+  public static final String CONNECTION_SSL_DEFAULT = "false";
+
+  public static final String CLIENT_AUTH_REQ_CONF = "elastic.https.client.auth.required";
+  private static final String CLIENT_AUTH_REQ_DOC = "Specify whether the client (connector) will "
+      + "be required to authenticate itself for SSL communication.";
+  public static final String CLIENT_AUTH_REQ_DEFAULT = "false";
+
   protected static ConfigDef baseConfigDef() {
     final ConfigDef configDef = new ConfigDef();
     addConnectorConfigs(configDef);
     addConversionConfigs(configDef);
+    ConfigDef sslConfigDef = new ConfigDef();
+    addClientSslSupport(sslConfigDef);
+    configDef.embed(
+        CONNECTION_SSL_CONFIG + ".", "HTTPS",
```

Comment:
Nit: Using "HTTPS" for a group name seems less useful than something like "Security" or "Secure Connections"